package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bitgo/unspents",
-  "version": "0.4.4-rc1",
+  "version": "0.4.5-rc1",
```

Comment:
Not sure how we are versioning this repo but if we are doing it according to semver the previous change added functions to the API so would technically be a minor version change.

Reply:
Okay